integrationtests/server-integration/third-party-server/pom.xml

```diff
@@ -0,0 +1,220 @@
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <target>
+                                        <echo message="unzipping tomcat server" />
+                                        <unzip src="${tomcat.zip}" dest="${project.build.directory}/tomcat" />
+                                        <pathconvert property="tomcat.unzip.dir">
+                                            <dirset dir="${project.build.directory}/tomcat" includes="apache-tomcat-*"/>
+                                        </pathconvert>
+                                        <move todir="${project.build.directory}/tomcat">
+                                            <fileset dir="${tomcat.unzip.dir}"/>
+                                        </move>
+                                        <copy file="${basedir}/src/test/resources/tomcat/tomcat-users.xml" todir="${project.build.directory}/tomcat/conf" overwrite="true" />
+                                    </target>
+                                </configuration>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>third-party-wildfly</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>infinispan.server.integration.launch</name>
+                    <value>wildfly</value>
+                </property>
+            </activation>
+            <properties>
+                <infinispan.server.integration.launch>wildfly</infinispan.server.integration.launch>
+                <jbossHome>${project.build.directory}/wildfly-${version.wildfly}</jbossHome>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-managed</artifactId>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>unpack</id>
+                                <phase>process-test-classes</phase>
```

Comment:
can you change the phase so I don't have to download the WF every time I compile Infinispan?

Reply:
It is downloading even when phase is `pre-integration-test`. Any idea?

Reply:
yes, add `<skip>${skipTests}</skip>` to the configuration :)

Reply:
Done